### PR TITLE
Loan Req fields

### PR DESCRIPTION
**Pull request by RossStewart**

### CODE REVIEW

File: src/Resources/Requisitions/RequisitionLineResource.cs [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 {
+    using System;
+
     public class RequisitionLineResource
     {
         public string TransactionCode { get; set; }
 
         public int Line { get; set; }
+
+        public DateTime? DateCancelled { get; set; }
```
Comment (lewisrenfrew) [inline src/Resources/Requisitions/RequisitionLineResource.cs:11]: should be a string?

File: src/Resources/Requisitions/RequisitionResource.cs [hunk 2]
```diff
@@ -26,6 +27,16 @@ public class RequisitionResource : HypermediaResource
 
         public string StorageType { get; set; }
 
+        public string Cancelled { get; set; }
+
+        public int? CancelledBy { get; set; }
+
+        public DateTime? DateCancelled { get; set; }
```
Comment (lewisrenfrew) [inline src/Resources/Requisitions/RequisitionResource.cs:34]: again should be a string

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 3]
```diff
@@ -1515,6 +1515,12 @@ private void BuildRequisitionHeaders(ModelBuilder builder)
             e.Property(r => r.ToLocationId).HasColumnName("TO_LOCATION_ID");
             e.Property(r => r.Document1Name).HasColumnName("DOC1_NAME");
             e.Property(r => r.PartNumber).HasColumnName("PART_NUMBER").HasMaxLength(14);
+            e.Property(r => r.Document1Name).HasColumnName("DOC1_NAME");
```
Comment (lewisrenfrew) [inline src/Persistence.LinnApps/ServiceDbContext.cs:1518]: this propery is alread mapped two lines above

Comment (RossStewart) [reply src/Persistence.LinnApps/ServiceDbContext.cs:1518]: Small typo, just fixing in next commit - meant to be the line.. Sorting dates too.

**Review by lewisrenfrew — APPROVED** [review]

**Pull request merged**